GiEnJul/ClientApp/src/components/admin/common/Datatable.tsx

```diff
@@ -20,7 +20,7 @@ import { GiverType } from "../overview/Giver";
 import useStyles from "./Styles";
 
 type Props = {
-  data: [GiverType] | [];
+  data: GiverType[] | [];
```

Comment:
The filename should be changed to reflect its usage properly. Given that Datatable is handling Givers it could be named GiverTable